### PR TITLE
testing pushing branch to github main

### PR DESCRIPTION
**Pull request by maxpeterson96**
@justinjchong @cegarcia96 @erinantoine1 

testing merge request

### CODE REVIEW

**Comment by cegarcia96** [comment]
Hey Max what's the purpose of the console log? Please explain

**Comment by justinjchong** [comment]
please change 'Axios' to 'axios'

File: client/src/components/QuestionsAnswers/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const message = 'hello this is a new message';
```
Comment (cegarcia96) [inline client/src/components/QuestionsAnswers/index.js:1]: message?

**Review by cegarcia96 — APPROVED** [review]
Looks good to me

**Review by justinjchong — COMMENTED** [review]
This is a review

**Review by justinjchong — APPROVED** [review]
I approve this pull

**Pull request merged**